tests/components/homekit/test_accessories.py

```diff
@@ -317,7 +318,8 @@ def test_home_bridge(hk_driver):
     assert serv.get_characteristic(CHAR_FIRMWARE_REVISION).value == __version__
     assert serv.get_characteristic(CHAR_MANUFACTURER).value == MANUFACTURER
     assert serv.get_characteristic(CHAR_MODEL).value == BRIDGE_MODEL
-    assert serv.get_characteristic(CHAR_SERIAL_NUMBER).value == BRIDGE_SERIAL_NUMBER
+    assert serv.get_characteristic(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4570585)